packages/label-wrapper/src/styles.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div<{errorTitle: string}>`
+  position: relative;
+
+  ${props => _ =>
+    props.errorTitle.length > 0 && {
+      '& * ': {
+        color: _.theme.colors.error.red,
+        borderColor: _.theme.colors.error.red
+      },
+      input: {
```

Comment:
i'm sure that actual input being child of this wrapper would want to decide about its style by itself - we're not making form validation UI here, just a method to add labels.

Reply:
I agree. Error handling should not be mixed with label.